sw/device/lib/testing/mock_mmio.h

```diff
@@ -141,6 +141,22 @@ Int ToInt(std::initializer_list<BitField> fields) {
  */
 inline internal::LittleEndianBytes LeInt(const char *bytes) { return {bytes}; }
 
+/**
+ * Returns an integer with the bit at offset set to 1 and all other bits to 0.
+ */
+template <typename Int>
+Int ToIntBit(Int offset) {
+  return (1 << offset);
+}
+
+/**
+ * Returns an integer with the bit at offset set to 0 and all other bits to 1.
+ */
+template <typename Int>
+Int ToIntBitInverted(Int offset) {
+  return ~(1 << offset);
+}
+
```

Comment:
I think these should go inside the `namespace mock_mmio`, probably the best place for them after
```
template <typename Int>
Int ToInt(std::initializer_list<BitField> fields)
```
and before
`}  // namespace internal`